components/ai_chat/resources/page/components/premium_suggestion/index.tsx

```diff
@@ -29,35 +52,41 @@ function PremiumSuggestion(props: PremiumSuggestionProps) {
 
   return (
     <div className={styles.boxPremium}>
-      <h4>{props.title}</h4>
-      <p>{props.description}</p>
-      <ul className={styles.featuresListing}>
-        <li>
-          <Icon name='check-normal' />
-          <span>
-            {getLocale('premiumFeature_1')}
-            {props?.verbose && <p>{getLocale('premiumFeature_1_desc')}</p>}
-          </span>
-        </li>
-        <li>
-          <Icon name='check-normal' />
-          <span>
-            {getLocale('premiumFeature_2')}
-            {props?.verbose && (
-              <p>{getLocale('premiumFeature_2_desc')}</p>
-            )}
-          </span>
-        </li>
-        <li className={styles.priceList}>
-          <span>{getLocale('premiumLabel')}</span>
-          <span className={styles.price}>
-            {pricingInfo}
-          </span>
-        </li>
-      </ul>
-      <div className={styles.actionsBox}>
+      <div className={styles.header}>
+        <h4>{props.title}</h4>
+        <p>{props.description}</p>
+      </div>
+      <div className={styles.features}>
+        <ul>
+          {featuresList.map((entry) => {
+            return (
+              <li>
+                <div className={styles.icon}>
+                  <Icon name={entry.icon} />
+                </div>
+                <span>
+                  {entry.title}
+                  <p>{entry.desc}</p>
+                </span>
+              </li>
+            )
+          })}
+        </ul>
+      </div>
+      {!loadTimeData.getBoolean('isMobile') && (
```

Comment:
It would be useful to have this switchable in Storybook. Probably, since loadTimeData is a platform-dependent control, we should put `isMobile` in the context provider

Reply:
isMobile is part of react context now and you can change it via storybook controls.